spec/integration_spec.rb

```diff
@@ -359,167 +355,6 @@
   end
 end
 
-describe 'Book' do
-  before do
-    Book.clear_index!(true)
-    Book.index(safe_index_uid('BookAuthor')).delete_all_documents
-    Book.index(safe_index_uid('Book')).delete_all_documents
-  end
-
-  it 'returns array of tasks on #ms_index!' do
-    moby_dick = Book.create! name: 'Moby Dick', author: 'Herman Melville', premium: false, released: true
-
-    tasks = moby_dick.ms_index!
-
-    expect(tasks).to contain_exactly(
-      a_hash_including('uid'),
-      a_hash_including('taskUid'),
-      a_hash_including('taskUid')
-    )
-  end
-
-  it 'indexes the book in 2 indexes of 3' do
```

Comment:
Moved to `options_spec.rb` since it tests whether or not the `:if` option is behaving correctly.